common/src/contract.ts

```diff
@@ -98,6 +98,10 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   token: ContractToken
   siblingContractId?: string
 
+  sportsStartTimestamp?: string
```

Comment:
I'd prefer these were in sth like 
```
sports?:{
startTs:string
...etc.
}
```
so that they would all be defined when sports is defined, otherwise only sports will be undefined.